README.markdown

```diff
@@ -16,17 +16,17 @@ $ heroku plugins:install heroku-kafka
 Create a heroku app with Kafka attached:
 
 ```
-$ heroku create
-$ heroku addons:create heroku-kafka:standard-0
-$ heroku kafka:wait
+$ heroku apps:create your-cool-app-name
+$ heroku git:remote -a your-cool-app-name
```

Comment:
This isn't necessary as the `heroku create` account sets this up for you.

Reply:
What?! TIL 🙇